src/os/bluestore/BlueFS.cc

```diff
@@ -546,14 +549,25 @@ uint64_t BlueFS::get_block_device_size(unsigned id) const
   return 0;
 }
 
+void BlueFS::drop_pending_release(uint64_t delta)
+{
+  ceph_assert(delta <= pending_release);
+  pending_release -= delta;
+  if (wait_pending_release && pending_release == 0) {
+    pending_release_cond.notify_all();
+  }
+}
+
 void BlueFS::handle_discard(unsigned id, interval_set<uint64_t>& to_release)
```

Comment:
This is offtopic, but we named it wrong. `handle_discard` suggest we are about to do discard, but in fact the discard is already done.

Reply:
Renamed.